README.md

```diff
@@ -206,17 +206,17 @@ Source code layout
 ---
 
 The Redis root directory just contains this README, the Makefile which
-actually calls the real Makefile inside the `src` directory, an example
-configuration for Redis and Sentinel. Finally you can find a few shell
+calls the real Makefile inside the `src` directory and an example
```

Comment:
The words `shell` and `calls` are not consecutive. It's actually `Makefile which calls`. Am I missing something?